### ISSUE TITLE
Slim always minified?

### ISSUE THREAD

**Issue by JacqsLabz** [bdkjones/CodeKit2#628]
No matter what I set for the Output style the html that the slim files create is always minified down to 1 line of code with no spaces / indents. 
For example:
`<head>`
      `<meta charset="utf-8"><meta name="robots" content="noindex" /><meta http-equiv="X-UA-Compatible" content="IE=edge">`
`<title>site ename</title>`
`<meta name="description" (etc)`
outputs as:
`<head><meta charset="utf-8"><meta name="robots" content="noindex" /><meta http-equiv="X-UA-Compatible" content="IE=edge"><title>site name</title><meta name="description" (etc)`

Or is that just how slim is supposed to work? I've never used slim or haml or jade before, and I'm trying to compare them before present one over the others to my boss.

**Comment by bdkjones** [comment]
Where are you selecting the output style -- project settings or on the individual file?

**Comment by JacqsLabz** [comment]
I was going on the individual file, but I just tried the project wide and it made no change. Anything that's html in the slim file compacts down to minified. 

I took the same file and actually converted it to proper slim instead of html, and then the indented option works. Though there's still some lines that start with | in the compiled html file...

Sorry if that's what slim is supposed to do. I haven't tried any other compilers to compare/contrast, and their documentation isn't the most helpful.
